e2e/app/component/base-help-sidebar.ts

```diff
@@ -94,4 +92,10 @@ export default abstract class BaseHelpSidebar extends Container {
       return false;
     }
   }
+
+  async waitUntilClose(): Promise<void> {
+    await super.waitUntilClose();
+    await this.page.waitForXPath(this.deleteIconXpath, { hidden: true });
```

Comment:
Just curious - why does waiting for the icon work, but the base `waitUntilClose` does not?

Reply:
`super.waitUntilClose` should works alone because it checks sidebar visibility in a generic way. Checking deleteIcon is extra and guarantee sidebar is not visible.